### ISSUE TITLE
Allow configuring a custom MQTT broker

### ISSUE THREAD

**Issue by swentel** [swentel/indigenous-android#216]
Follow up to #214 

For custom servers:

- allow adding a custom uri, and optionally username and password
- subscribe to something - need to figure out what exactly this is

**Comment by dshanske** [comment]
A topic, perhaps indigenous ?

**Comment by swentel** [comment]
Yep makes sense. To configure I currently see following input fields:

- uri
- login (optional)
- password (optional)

That should do it right ? (apart then for listeners of course)

**Comment by dshanske** [comment]
Port. SSL uses a different port than non ssl

**Comment by dshanske** [comment]
That might be in the Uri

**Comment by swentel** [comment]
List of brokers I could test with: https://mntolia.com/10-free-public-private-mqtt-brokers-for-testing-prototyping/

**Comment by swentel** [comment]
Gave this some more thoughts and I'm not going to support this for now.

Pull requests are welcome, but this one is so rare that it's a bit overkill to support this.

**Comment by swentel** [comment]
Reopening, there's time to implement this!
